source/_integrations/google_assistant.markdown

```diff
@@ -109,6 +109,8 @@ If you want to support active reporting of state to Google's server (configurati
 
 ### Enable Local Fulfillment
 
+If you want to use local fulfillment Home Assistant must be configured to not use HTTPS. If you need to serve Home Assistant over HTTPS consider using a reverse proxy.
```

Comment:
I think this information can be confusing with what we have at the top of the page:  `To use Google Assistant, your Home Assistant configuration has to be externally accessible with a hostname and SSL certificate.`

Reply:
It certainly is confusing, which is why I am trying to bring clarity, but maybe it needs to be a little more absolute to make it less confusing. I'll make some more changes.